catalog/core/catalog-core-commons/src/main/java/ddf/util/XPathHelper.java

```diff
@@ -70,7 +70,7 @@ public XPathHelper() {
             this.getClass().getClassLoader());
     dbf.setNamespaceAware(true);
     tf =
-        TransformerFactory.newInstance(
+        XML_UTILS.getSecureXmlTransformerFactory(
```

Comment:
@lessarderic I'm used to only seeing strings and primitives use the ALL_CAPS names. I prefer to see static objects use camel case. Maybe that is an additional refactoring we can make here?
`xmlUtils`

Reply:
I believe this is checkstyle enforced for static references? I could be wrong

Reply:
That is correct. https://github.com/codice/ddf-support/blob/master/support-checkstyle/src/main/resources/checkstyle-enforced.xml#L65

Reply:
Yep, all `static final` member variables need to be all caps. Sorry @ahoffer :-)